contract/models/account_analytic_account.py

```diff
@@ -29,7 +29,12 @@ class AccountAnalyticAccount(models.Model):
         inverse_name='analytic_account_id',
         copy=True,
     )
-    date_start = fields.Date(default=fields.Date.context_today)
+    date_start = fields.Date(
+        default=fields.Date.context_today,
+        string='Date Start')
```

Comment:
[Please add a trailing comma and put the `)` on next line](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms)

[More info - note about version control](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)